.github/workflows/check_pr.yml

```diff
@@ -0,0 +1,35 @@
+name: Check branch conformity
+on:
+  pull_request:
+      types: [opened, synchronize, reopened]
```

Comment:
Hm, [that's the default](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request) actually, so how come we specify it again?

> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened

Reply:
Yeap, that was me toying with it to understand why it wasn't triggering builds. I put it back on place.